### PR TITLE
Translate "CVE HTTP response splitting in CGI" with Ruby released (ko)

### PR DESCRIPTION
**Pull request by riseshia**
:link: #2818

Translation CVE-2021-33621 with Ruby release which includes it.

Actual diff

- CVE-2021-33621: HTTP response splitting in CGI -> 620abe7
- Ruby 2.7.7, 3.0.5, 3.1.3 released -> 8118a0a

### CODE REVIEW

File: ko/news/_posts/2022-11-22-http-response-splitting-in-cgi-cve-2021-33621.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+---
+
+HTTP 응답 분할 취약점에 대한 보안 수정을 포함하는 cgi gem 버전 0.3.5, 0.2.2, 0.1.0.2를 릴리스했습니다.
+이 취약점은 CVE 번호 [CVE-2021-33621](https://nvd.nist.gov/vuln/detail/CVE-2021-33621)로 등록되어 있습니다.
+
+## 세부 내용
+
+애플리케이션이 cgi gem을 사용해서 신뢰할 수 없는 사용자 입력으로부터 HTTP 응답을 생성할 때, 공격자는 악의 있는 HTTP 헤더, 본문을 삽입할 수 있습니다.
+
+또한, `CGI::Cookie` 객체의 내용이 올바른지 검사 되지 않았습니다. 애플리케이션이 사용자 입력으로부터 `CGI::Cookie` 객체를 생성할 때, 공격자는 `Set-Cookie` 헤더에 유효하지 않은 속성을 주입할 수 있습니다. 이러한 애플리케이션은 일반적이지 않습니다만, 예방 차원에서 `CGI::Cookie#initialize`가 인수를 검사하도록 변경했습니다.
```
Comment (yous) [inline ko/news/_posts/2022-11-22-http-response-splitting-in-cgi-cve-2021-33621.md:18]:
```suggestion
또한, `CGI::Cookie` 객체의 내용이 올바른지 제대로 검사되지 않았습니다. 애플리케이션이 사용자 입력으로부터 `CGI::Cookie` 객체를 생성할 때, 공격자는 `Set-Cookie` 헤더에 유효하지 않은 속성을 주입할 수 있습니다. 이러한 애플리케이션은 일반적이지 않습니다만, 예방 차원에서 `CGI::Cookie#initialize`가 인수를 검사하도록 변경했습니다.
```

File: ko/news/_posts/2022-11-24-ruby-2-7-7-released.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
+---
+layout: news_post
+title: "Ruby 2.7.7 릴리스"
+author: "usa"
+translator: "shia"
+date: 2022-11-24 12:00:00 +0000
+lang: ko
+---
+
+Ruby 2.7.7가 릴리스되었습니다.
```
Comment (yous) [inline ko/news/_posts/2022-11-24-ruby-2-7-7-released.md:10]:
```suggestion
Ruby 2.7.7이 릴리스되었습니다.
```

File: ko/news/_posts/2022-11-24-ruby-2-7-7-released.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+이 릴리스는 보안 수정을 포함합니다.
+자세한 사항은 아래 글을 확인해 보세요.
+
+* [CVE-2021-33621: CGI에서의 HTTP 응답 분할]({%link ko/news/_posts/2022-11-22-http-response-splitting-in-cgi-cve-2021-33621.md %})
+
+이 릴리스는 몇몇 빌드 문제 수정을 포함합니다. 이는 이전 버전들과 호환성 문제를 일으키지 않습니다.
```
Comment (yous) [inline ko/news/_posts/2022-11-24-ruby-2-7-7-released.md:17]:
```suggestion
이 릴리스는 몇몇 빌드 문제 수정을 포함합니다. 이는 이전 버전과 호환성 문제를 일으키지 않습니다.
```

File: ko/news/_posts/2022-11-24-ruby-3-1-3-released.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
+---
+layout: news_post
+title: "Ruby 3.1.3 릴리스"
+author: "nagachika"
+translator: "shia"
+date: 2022-11-24 12:00:00 +0000
+lang: ko
+---
+
+Ruby 3.1.3가 릴리스되었습니다.
```
Comment (yous) [inline ko/news/_posts/2022-11-24-ruby-3-1-3-released.md:10]:
```suggestion
Ruby 3.1.3이 릴리스되었습니다.
```

File: ko/news/_posts/2022-11-24-ruby-3-1-3-released.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+
+이 릴리스는 보안 수정을 포함합니다.
+자세한 사항은 아래 글을 확인해 보세요.
+
+* [CVE-2021-33621: CGI에서의 HTTP 응답 분할]({%link ko/news/_posts/2022-11-22-http-response-splitting-in-cgi-cve-2021-33621.md %})
+
+이 릴리스는 Xcode 14와 macOS 13(Ventura)에서의 빌드 실패에 대한 수정을 포함합니다.
+자세한 사항은 [관련 티켓](https://bugs.ruby-lang.org/issues/18912)을 확인해 주세요.
+
+자세한 사항은 [커밋 로그](https://github.com/ruby/ruby/compare/v3_0_4...v3_0_5)를 확인해 주세요.
```
Comment (yous) [inline ko/news/_posts/2022-11-24-ruby-3-1-3-released.md:20]:
```suggestion
자세한 사항은 [커밋 로그](https://github.com/ruby/ruby/compare/v3_1_2...v3_1_3)를 확인해 주세요.
```

**Comment by riseshia** [comment]
🙈

**Review by yous — APPROVED** [review]
Thanks!

**Pull request merged**